src/steps/Microsoft.DotNet.UpgradeAssistant.Steps.Razor/Microsoft.DotNet.UpgradeAssistant.Steps.Razor.csproj

```diff
@@ -12,8 +12,7 @@
     <PackageReference Include="DiffPlex" />
     <PackageReference Include="Microsoft.AspNetCore.Mvc.Razor.Extensions" />
     <PackageReference Include="Microsoft.AspNetCore.Razor.Language" />
-    <PackageReference Include="Microsoft.Bcl.Hashcode" />
     <PackageReference Include="Microsoft.CodeAnalysis.Razor" />
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" />
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" PrivateAssets="all" />
```

Comment:
Why not copy these assets to output?

Reply:
It gets filtered out as it's in the output already, but reverting this